schematic/store/synapse.py

```diff
@@ -100,23 +101,53 @@ def __init__(
 
         self._query_fileview()
 
+    def _purge_synapse_cache(self):
+        '''
+        Purge synapse cache if it exceeds 7 GB
+        '''
+        # define the root directory
+        # set maximum synapse cache allowed
+        # set maximum ephemeral stroage allowed on AWS
+        root_dir = "/var/www/.synapseCache/"
+        maximum_storage_allowed_cache_gb = 7
+        maximum_storage_allowed_cache_bytes = convert_gb_to_bytes(maximum_storage_allowed_cache_gb)
+        total_ephemeral_storag_gb = 20
+        total_ephemeral_storage_bytes = convert_gb_to_bytes(total_ephemeral_storag_gb)
```

Comment:
Nit: You technically have no use for these if the `root_dir` doesn't exist, so it may make sense to move them into the if statement.